global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "9.0.303",
+    "version": "8.0.117",
```

Comment:
Why did you downgrade the version. Leave it alone. It is not related to the task